### PR TITLE
Dnvglnozzle

### PR DESCRIPTION
**Pull request by smolvik1**
Have a look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <img src="https://raw.githubusercontent.com/equinor/pysand/master/resources/logo.png" align="center" title="pySand"/>
 
 [![PyPI](https://img.shields.io/pypi/v/pysand)](https://pypi.org/project/pysand/)
-[![Build status](https://travis-ci.org/equinor/pysand.svg?master)](https://travis-ci.org/equinor/pysand)
+[![Build status](https://travis-ci.org/equinor/pysand.svg?branch=master)](https://travis-ci.org/equinor/pysand)
```
Comment (knudsvik) [inline README.md]: What is the rationale? I think the result is the same?

File: pysand/erosion.py [hunk 2]
```diff
@@ -394,6 +392,36 @@ def choke_gallery(v_m, rho_m, mu_m, GF, D, d_p, R_c, gap, H, material='cr_37_tun
...
+    :return: Relative erosion rate [mm/ton]
+    """
+    # Input validation
+    kwargs = {'v_m': v_m, 'd_p': d_p, 'GF': GF, 'At': At}
+    if validate_inputs(**kwargs):
+        return np.nan
+    
+    # Stricter particle size validation
+    if (d_p > 0.6):
+            exc.FunctionInputFail('Particle diameter, d_p, is highter than CFD-study boundary (0.6 mm).')
```
Comment (knudsvik) [inline pysand/erosion.py]: This function input fail is not tested. Should it be?

File: pysand/erosion.py [hunk 3]
```diff
@@ -394,6 +392,36 @@ def choke_gallery(v_m, rho_m, mu_m, GF, D, d_p, R_c, gap, H, material='cr_37_tun
...
+    :param material: Material exposed to erosion, default = 'duplex'. For others, run: materials()
+    :return: Relative erosion rate [mm/ton]
+    """
+    # Input validation
+    kwargs = {'v_m': v_m, 'd_p': d_p, 'GF': GF, 'At': At}
+    if validate_inputs(**kwargs):
+        return np.nan
+    
+    # Stricter particle size validation
+    if (d_p > 0.6):
```
Comment (knudsvik) [inline pysand/erosion.py]: This is input validation which maybe should be done in the validation function instead.

File: pysand/erosion.py [hunk 4]
```diff
@@ -394,6 +392,36 @@ def choke_gallery(v_m, rho_m, mu_m, GF, D, d_p, R_c, gap, H, material='cr_37_tun
...
+            exc.FunctionInputFail('Particle diameter, d_p, is highter than CFD-study boundary (0.6 mm).')
+
+    # Constants:
+    def c1(d_p):
+        corr = 8.33 * d_p**3 - 29.2 * d_p**2 + 22.8 * d_p + 1
+        return corr
+    C1 = c1(d_p) # Model geometry factor
```
Comment (knudsvik) [inline pysand/erosion.py]: Why don't you just write `C1 = 8.33 * d_p**3 - 29.2 * d_p**2 + 22.8 * d_p + 1`? The function only reduces readability?

File: README.md (hunk 1 above)
Comment (smolvik1) [reply README.md]: I noticed when this branch was failing build tests, that the sticker on master-branch readme.md also showed build failing. When changing to branch=master, the logo showed passing.

**Comment by knudsvik** [comment]
Resolves #41

**Pull request merged**